doc/using/configuration.chapter.md

```diff
@@ -10,19 +11,11 @@ Nix comes with certain defaults about what packages can and cannot be installed,
 
 -   The package has known security vulnerabilities but has not or can not be updated for some reason, and a list of issues has been entered in to the package's `meta.knownVulnerabilities`.
 
-Note that all this is checked during evaluation already, and the check includes any package that is evaluated. In particular, all build-time dependencies are checked. `nix-env -qa` will (attempt to) hide any packages that would be refused.
+Note that all this is checked during evaluation already, and the check includes any package that is evaluated.
+In particular, all build-time dependencies are checked.
 
-Each of these criteria can be altered in the nixpkgs configuration.
+Each of these criteria can be altered in the Nixpkgs configuration.
 
-The nixpkgs configuration for a NixOS system is set in the `configuration.nix`, as in the following example:
-
-```nix
-{
-  nixpkgs.config = {
-    allowUnfree = true;
-  };
-}
-```
 
 However, this does not allow unfree software for individual users. Their configurations are managed separately.
```

Comment:
This line should also be removed, it was part of the explanation for the configuration you removed.